aml-development/frontend
# feat(subscriptions): allow user to edit tag subscriptions

AMLOS-696

**Description**
As a user, I would like the ability to subscribe/unsubscribe from tags.

**Acceptance Criteria**
A user can subscribe to a tag for notifications
A user can unsubscribe to tag notifications from the user profile page.

**How to test code**
Pull `tag_subscriptions` from `frontend`
If https://github.com/aml-development/ozp-backend/pull/434 is not merged, pull `subscription_filter` from `ozp-backend`

- Log in and go to the user profile edit page
- Add/remove tag subscriptions using the drop-down in the _Preferences_ section.
- The list of tags should filter down as the user searches.  The list is limited to 24 results and there should be a 300ms input delay before performing an API call.
- Saving the profile should update the tag subscriptions (and everything else in the profile) accordingly.
